### PR TITLE
Add github workflows to release helm chart

### PR DESCRIPTION
**Pull request by yongfengdu**
* manual-freeze-tag.yaml to run on release branch, modify version and tags
* manual-helm-cd-workflow.yaml to trigger tests
* manual-release-charts.yaml to release the helm charts to public repo

## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Issues

#561 

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: .github/workflows/manual-release-charts.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+            git config --global user.name "NeuralChatBot"
+            git config --global user.email "grp_neural_chat_bot@intel.com"
+            git remote set-url origin https://NeuralChatBot:"${{ secrets.ACTION_TOKEN }}"@github.com/opea-project/GenAIInfra.git
+
+            git status
+            git add .
+            git commit -s -m "Add release ${{ inputs.version }} helm charts"
+            git push
+          else
```
Comment (poussa) [inline .github/workflows/manual-release-charts.yaml:81]: else branch not needed (just print), is it?

File: .github/workflows/manual-release-charts.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          if [ "${{ inputs.artifacthub }}" = "true" ]; then
+            echo "Push ${{ inputs.version }} charts to charts-release branch"
+            helm repo index .
+            # Insert copyright to avoid warnings
+            sed -i '1s/^/# Copyright (C) 2024 Intel Corporation\n# SPDX-License-Identifier: Apache-2.0\n\n/' index.yaml
+
+            git config --global user.name "NeuralChatBot"
+            git config --global user.email "grp_neural_chat_bot@intel.com"
+            git remote set-url origin https://NeuralChatBot:"${{ secrets.ACTION_TOKEN }}"@github.com/opea-project/GenAIInfra.git
+
```
Comment (poussa) [inline .github/workflows/manual-release-charts.yaml:76]: What are these Neural Chat Bot things?

Comment (yongfengdu) [reply .github/workflows/manual-release-charts.yaml:76]: This is the account setup for GH auto commit, inherited from other GH actions. I don't know why the name, just keep consistent with others.

File: .github/workflows/manual-release-charts.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "1.1.0"
+        description: "Release Version"
+        required: true
+        type: string
+      artifacthub:
```
Comment (mkbhanda) [inline .github/workflows/manual-release-charts.yaml:14]: please comment or remove this artifacthub section. We will publish to one place only per earlier discussion. Assume you leave it here and then do not publish per discussion, someone will think its a bug.

**Review by mkbhanda — APPROVED** [review]
Thank you @yongfengdu ! LGTM

**Review by poussa — APPROVED** [review]

**Pull request merged**